apps/website/src/pages/clips/submit-clip.tsx

```diff
@@ -0,0 +1,128 @@
+import type { NextPage } from "next";
+import Image from "next/image";
+import type { FormEvent } from "react";
+import { useState } from "react";
+import { useSession } from "next-auth/react";
+
+import { useRouter } from "next/router";
+import Meta from "@/components/content/Meta";
+import Section from "@/components/content/Section";
+import Heading from "@/components/content/Heading";
+
+import leafLeftImage2 from "@/assets/floral/leaf-left-2.png";
+import leafRightImage2 from "@/assets/floral/leaf-right-2.png";
+
+import { Fieldset } from "@/components/shared/form/Fieldset";
+import { TextField } from "@/components/shared/form/TextField";
+import { Button } from "@/components/shared/Button";
+import type { ClipSubmitInput } from "@/server/db/clips";
+import { trpc } from "@/utils/trpc";
+import { MessageBox } from "@/components/shared/MessageBox";
+import IconLoading from "@/icons/IconLoading";
+import { PageNavigation } from "@/components/shared/PageNavigation";
+
+const SubmitClipPage: NextPage = () => {
+  const add = trpc.clips.addClip.useMutation();
+  const router = useRouter();
+  const [error, setError] = useState<string | null>(null);
+  const [submitting, setSubmitting] = useState(false);
+
+  const clipsNavigationItems = [
+    {
+      href: "/clips/",
+      label: "Clips",
+      exact: true,
+    },
+    {
+      href: "/clips/submit-clip",
+      label: "Submit Clip",
+    },
+  ];
```

Comment:
Instead of repeating the items maybe create a component wrapping the `PageNavigation` in its own file and co-locate the items array there? Exporting the items from one file and reusing them in the other can also work (see comment about the relative import for show and tell).